editor/js/test.js

```diff
@@ -39,14 +39,19 @@ const puppeteer = require('puppeteer');
     const url = tests[i];
     await page.goto(url);
     await page.waitFor('body[ready=true]');
-    await page.evaluate(async url => {
+    testResult = await page.evaluate(async function (url) {
```

Comment:
Changed the next few lines to clarify things. I kept the variable for now, but with these new changes, would a lambda be better?

Reply:
I think the lambda that was here before is concise and more readable. Why did you change it?

Generally speaking, it is helpful to me as a reviewer if a PR has a coherent purpose. In this case, I would have considered making one PR to add validation, another to make stylistic changes, a third to address proto field presence, etc.  Just something to think about next time.

Reply:
No (real good) reason, I just thought this might be more readable. I'd be happy to change (although the PR's been merged since...).

About the PR, definitely -- I should've been more careful about that. Thanks for the tip.